packages/example/src/pages/components/Accordion.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Accordion
+---
+import { AccordionGroup, Accordion, AccordionItem } from "../../../../gatsby-theme-carbon";
```

Comment:
Yarn workspace ensures we use the component from the theme without a direct import.

```suggestion
import { AccordionGroup, Accordion, AccordionItem } from "gatsby-theme-carbon";
```